scripts/ci/test_plan.py

```diff
@@ -125,6 +125,11 @@ def process(self):
         if not self.platforms:
             self.find_archs()
             self.find_boards()
+        else:
+            for file in self.modified_files:
+                if file.startswith("boards/") or file.startswith("dts/"):
```

Comment:
Same comment here

```suggestion
                if file.startswith(("boards/", "dts")):
```

Reply:
Maybe I should add `flake8-pie` to #81070, see https://docs.astral.sh/ruff/rules/multiple-starts-ends-with/ 😉